src/packages/BalancedJs/contracts/sICX.ts

```diff
@@ -32,6 +32,20 @@ export default class sICX extends Contract {
     return this.callIconex(payload);
   }
 
+  async collateralDeposit(value: number): Promise<ResponseJsonRPCPayload> {
```

Comment:
Sorry @tuan-tl 
I can't understand what the function does?
For your demo, I thought it is the transfer function for peer-to-peer sICX transfer. 
For your function implementation, It doesn't do that.
Could you explain more?

Reply:
Can you please take a look back at this linked issue https://github.com/balancednetwork/balanced-network-interface/issues/70?